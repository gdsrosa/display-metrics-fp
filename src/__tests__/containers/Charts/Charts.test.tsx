import { describe, it, expect, vi } from 'vitest';
import { render } from '@testing-library/react';

import ChartsContainer from '@/containers/Charts';

vi.mock('react-chartjs-2');

describe('Charts Container', () => {
  it('should render charts with data that is passed', async () => {
    const { container } = render(<ChartsContainer />);

    expect(container).toMatchSnapshot();
  });
});
