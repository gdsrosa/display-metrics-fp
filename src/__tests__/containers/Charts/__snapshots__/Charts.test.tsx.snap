// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Charts Container > should render charts with data that is passed 1`] = `
<div>
  <div
    class="sc-bgqQPU kOUxmS"
    data-testid="card"
  >
    <h3
      class="sc-ewnpUy kDYXBG"
    />
    <div
      class="sc-gTRrcP huiFUI"
    >
      <div
        class="sc-fFGkvd pEWOf"
        data-testid="charts"
      />
    </div>
  </div>
</div>
`;
